### PR TITLE
fix: hover card name breaking point

### LINKED ISSUE #1753: Limit lenght of name on hover card
https://nft-gallery-5ci.pages.dev/spotlight

image

### PR DESCRIPTION
**Pull request by kkukelka**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇  _ Do a quick check before the merge.

### PR type
- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

### What's new?
- [x] PR closes #1753
- [x] solution first seems overkill, but actually handles all cases elegantly. **there is no need to truncate!** if words in names exceed parents bounds, it now gets broken on any point -> names with multiple words still get rendered as is
Screenshot 2022-01-10 at 19 57 16
Screenshot 2022-01-10 at 19 57 38
Screenshot 2022-01-10 at 19 57 06

### Before submitting Pull Request, please make sure:
- [ ] My contribution builds **clean without any errors or warnings**
- [ ] I've merged recent default branch -- **main-nuxt** and I've no conflicts
- [ ] I've tried respect high code quality standards
- [ ] I've didn't break any original functionality
- [ ] I've posted screenshot of demonstrated change in this PR

### Optional
- [x] I've tested it at </rmrk/u/EtF8MpQPXrqAtrwvuE6Ni4BvBMNPMXyxTug6WkzuV5utB1e>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases

### Had issue bounty label ?
- [x] Fill up your KSM address: [Payout](https://kodadot.xyz/transfer/?target=EqdyzrzVmeHwMdMwvPeCMnNdbuQDbD3YrjY93xq9Ln3jUGW)

### Community participation
- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot
- [x] My fix has changed **something** on UI, a screenshot is best to understand changes for others.

### CODE REVIEW

**Comment by vikiival** [comment]
> solution first seems overkill

what if we just use break all always?

**Comment by kkukelka** [comment]
> > solution first seems overkill
> 
> what if we just use break all always?

then normal names "Romero the Degenerate Artist" will break at very weird points
like:
```
Romero the Dege
nerate Artist
```

**Comment by roiLeo** [comment]
Why don't we use `overflow-wrap: break-word;` ? css only solution

**Comment by kkukelka** [comment]
> Why don't we use `overflow-wrap: break-word;` ? css only solution

because this doesn't solve the issue of unusually long words
Screenshot 2022-01-11 at 11 32 27
Screenshot 2022-01-11 at 11 34 57

**Comment by roiLeo** [comment]
I think this should work with a fixed width on container.

How do you manage to block tippy container to take screenshot?

**Comment by vikiival** [comment]
> How do you manage to block tippy container to take screenshot?

on the tippy element enable :hover (in chrome dev console)

**Comment by kkukelka** [comment]
> I think this should work with a fixed width on container.
> 
> How do you manage to block tippy container to take screenshot?

you sir are totally right!
discrete width works.
this is incredibly good to know 🎉 😘

File: components/shared/IdentityPopover.vue [hunk 1]
```diff
@@ -159,4 +159,12 @@ export default class IdentityPopover extends mixins(PrefixMixin) {
 .copy-icon {
   cursor: pointer;
 }
+
+.identity-info {
+  width: 75%
+}
```
Comment (roiLeo) [inline components/shared/IdentityPopover.vue:165]: hmmmmmmmmmm 🎲

**Review by roiLeo — APPROVED** [review]
lgtm, former code had fixed width that could be resolved using columns (first row).

File: components/shared/IdentityPopover.vue (hunk 1 above)
Comment (kkukelka) [reply components/shared/IdentityPopover.vue]: done via columns now

**Review by vikiival — APPROVED** [review]
No more 18-word-length magick

**Pull request merged**